### ISSUE TITLE
How to download the AES70-4 AES70 JSON protocol document?

### ISSUE THREAD

**Issue by ioniconline** [OCAAlliance/OCAMicro#27]
Would you please tell me how to download these documents?

AES70-4 AES70 JSON protocol. Text + UML document that defines OCP.2, a JSON protocol for using AES70 over IP networks. 

Adaptations AES70-21 AES67 Adaptation. This standard will specify the use of AES70 connection management for AES67 stream transport connections. 

AES70-22 MILAN Adaptation. This standard will specify the use of AES70 connection management for MILAN stream transport connections. 

Thanks.

**Comment by tljdebrouwer** [comment]
Hi, 

All standard documents can be downloaded via AES, for example for the framework see https://www.aes.org/publications/standards/search.cfm?docID=101

I'm not sure about the status of these documents, maybe @JeffBerryman can you help on that?

**Comment by JeffBerryman** [comment]
Hi.   The documents you mention are not published yet.  I guess you may have seen them listed in one of my recent talks at ISE.  I'm sorry I didn't make their status clear.  We expect to publish AES70-21 and -22 around August of this year, and AES70-4 a few months after that.   Please feel free to contact me directly (ja.beryman@us.bosch.com) if you'd like to discuss this further.

**Comment by ioniconline** [comment]
Thanks for reply

**Comment by ioniconline** [comment]
Have you released AES70-4 json protocol?

**Comment by JeffBerryman** [comment]
No, we haven't released it yet.  Our current workplan is to release it in the Fall of 2024.

Because it's been a while since we worked on the JSON version, we have decided to do a brief review of requirements to make sure we'll be releasing something people will be able use effectively.  Can you tell me anything about your use case?

Thanks
Jeff Berryman
chair, OCA Alliance Technical Committee
***@***.***
+1 952 457 5445 (timezone UTC-4 - Eastern US)

**Comment by ioniconline** [comment]
Hi　Jeff，　have you released AES70-4 json protocol now?

**Comment by JeffBerryman** [comment]
Hi!  No, not yet.  In March or April, the draftAES70-4  standard will be posted on the AES webite for public comment.  At that time, anyone will be able to download and read the draft.
The period for public comment is six weeks long.  At the end of that period, and after public comments (if any) have been resolved, the draft will be published as the official standard.
cheers // Jeff
